src/database/common.c

```diff
@@ -225,24 +219,24 @@ void db_init(void)
 	int dbversion = db_get_int(db, DB_VERSION);
 	if(dbversion < 1)
 	{
-		logg("Database not available, please ensure the database is unlocked when starting pihole-FTL !");
+		log_warn("Database not available, please ensure the database is unlocked when starting pihole-FTL !");
 		dbclose(&db);
 		return;
 	}
 	else
 	{
-		logg("Database version is %i", dbversion);
+		log_info("Database version is %i", dbversion);
 	}
 
 
 	// Update to version 2 if lower
 	if(dbversion < 2)
 	{
 		// Update to version 2: Create counters table
-		logg("Updating long-term database to version 2");
+		log_info("Updating long-term database to version 2");
 		if (!create_counter_table(db))
 		{
-			logg("Counter table not initialized, database not available");
+			log_warn("Counter table not initialized, database not available");
```

Comment:
Above you assigned `database not available` to `ERR`. I agree.
```suggestion
			log_err("Counter table not initialized, database not available");
```